core/src/main/java/org/mskcc/oncotree/utils/TumorTypesUtil.java

```diff
@@ -51,7 +51,7 @@
 public class TumorTypesUtil {
 
     private final static Logger logger = Logger.getLogger(TumorTypesUtil.class);
-    public final static String TSV_HEADER = "primary\tsecondary\ttertiary\tquaternary\tquinternary\tmetamaintype\tmetacolor\tmetanci\tmetaumls";
+    public final static String TSV_HEADER = "primary\tsecondary\ttertiary\tquaternary\tquinternary\tmetamaintype\tmetacolor\tmetanci\tmetaumls\thistory";
```

Comment:
tumortype.txt expansion : new column for history (maybe this should match previous_code from the tumorTypes api?

Reply:
oh .. it already does match .. because we are staying consistent with the current API by reusing the field "history". So never mind.